src/chainparams.cpp

```diff
@@ -337,10 +349,11 @@ class CRegTestParams : public CTestNetParams {
         BOOST_STATIC_ASSERT(equihash_parameters_acceptable(N, K));
```

Comment:
Upstream did not do so (then again, they did remove alert keys later), and I don't see any benefit of using the same alert key for testnet and regtest. Removing that usage doesn't break any tests, and testing that would be better done with a dedicated (and hard-coded) alert private key.